### DISCUSSION TITLE
Gatsby Cloud overwriting reverse proxy url's

### DISCUSSION THREAD

**Discussion by lanelps** [gatsbyjs/gatsby#35521] [Help]
I have **two** Gatsby sites hosted on Gatsby Cloud. Site `x` has a url rewrite/ reverse proxy to the home page of site `y`.

So for example `https://gatsby-site-x.gatsby.io/test` should show `https://gatsby-site-y.gatsby.io/` but at `https://gatsby-site-x.gatsby.io/test`.

This works, but the `https://gatsby-site-x.gatsby.io/test` url is quickly changed back to it's root url `https://gatsby-site-x.gatsby.io` but still has site `y` showing.

I think this cause by Gatsby Cloud but or maybe just the nature of  Gatsby sites themselves. How do I prevent this from happening?

**Comment by LekoArts** [comment]
For Gatsby Cloud related questions please go here: https://www.gatsbyjs.com/support/
